src/main/java/edu/kit/kastel/sdq/artemis4j/grading/TestResult.java

```diff
@@ -19,6 +19,10 @@ public TestResult(FeedbackDTO dto) {
         this.dto = dto;
     }
 
+    public long getId() {
+        return id;
+    }
+
```

Comment:
What is this new method used for?

Reply:
It was just that we can access the ID directly.